R/pkg/inst/worker/worker.R

```diff
@@ -50,7 +50,7 @@ compute <- function(mode, partition, serializer, deserializer, key,
     } else {
       # Check to see if inputData is a valid data.frame
       stopifnot(deserializer == "byte" || deserializer == "arrow")
-      stopifnot(class(inputData) == "data.frame")
+      stopifnot(is.data.frame(inputData))
```

Comment:
In Arrow 0.14, seems `as.data.frame(arrow_batch)` returns non-naive R DataFrame ("tbl_df tbl data.frame"). So, direct class comparison doesn't look working here.